lib/node_runner.go

```diff
@@ -17,7 +19,7 @@ import (
 	logging "github.com/inconshreveable/log15"
 
 	"boscoin.io/sebak/lib/common"
-	"boscoin.io/sebak/lib/network"
+	// "boscoin.io/sebak/lib/network"
```

Comment:
when you finish your PR then, please remove this kind of unnecessary changes

Reply:
@kfangw OK I will. Thanks.